src/Command/FrontendCommands.php

```diff
@@ -150,8 +151,15 @@ private function frontendLink()
     * "extra" section of DKAN's composer.json. If you want to specify a
     * different tag or branch, or different repo entirely, run "dktl
     * frontend:get" first and specify the --ref and/or --url options.
+    *
+    * @param array $opts
+    *   Options array.
+    * @option theme
+    *   Composer requirement for theme to download and enable.
+    *   Pass --no-theme to skip theme installation. Defaults to
+    *   "getdkan/dkan_js_frontend_bartik".
     */
-    public function frontendInstall()
+    public function frontendInstall($opts = ['theme' => true])
```

Comment:
```suggestion
    public function frontendInstall(array $opts = ['theme' => self::FRONTEND_THEME])
```
I think replacing `true` with the default theme constant would make the method signature a bit more readable.